Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Utilities/InteractibleHighlight.cs

```diff
@@ -18,15 +18,15 @@ public class InteractableHighlight : BaseFocusHandler
         [Flags]
         public enum HighlightedMaterialStyle
         {
-            None = 0,
+            None = 0 << 0,
             /// <summary>
             /// A highlight to indicate focus.
             /// </summary>
-            Highlight = 1,
+            Highlight = 1 << 0,
```

Comment:
this looks really weird... shouldn't it be
Highlight = 1 << 0,
Overlay = 1 << 1,

Reply:
Isn't that what it is?